purchase_fop_shipping/models/partner.py

```diff
@@ -10,4 +10,5 @@ class ResPartner(models.Model):
     _inherit = 'res.partner'
 
     fop_shipping = fields.Float(
-        'FOP shipping', help='Min purchase order amount for FOP shipping',)
+        'FOP shipping', help='Min purchase order amount for FOP shipping',
+        company_dependent=True)
```

Comment:
@Cedric-Pigeon Apart of that, IMHO this field should use decimal precision (and a float compare in _compute_fop_shipping_reached)

Reply:
@rousseldenis Done